packages/cli/src/hooks/analytics/identifier.spec.ts

```diff
@@ -120,26 +107,24 @@ describe('identifier', () => {
       expect(mockSetIdentity).toHaveBeenCalledWith('region', 'us');
     });
 
-    it('should set is_internal_user to true', () => {
+    it('should set the user ID', async () => {
+      const identity = await new Identifier().getIdentity();
```

Comment:
I'd refactor identity in the describe scope, and do a beforeEach with the assignation to `new Identifier().getIdentity();`, given that you do it for every test afaik